ui/vite.config.ts

```diff
@@ -7,5 +7,14 @@ export default defineConfig({
   plugins: [react(), viteTsconfigPaths()],
   build: {
     outDir: "./build"
+  },
+  server: {
+    proxy: {
+      "/api": {
+        target: "http://0.0.0.0:8089",
```

Comment:
Can we use localhost?.. it'll never listen on 0.0.0.0

Reply:
Changed to 127.0.0.1 because "localhost" resolves to IPv6, and that doesn't work